packages/gestalt/src/ButtonSocial.css

```diff
@@ -0,0 +1,35 @@
+.social {
+  align-items: center;
+  border-radius: var(--rounding-pill, 999px);
+  cursor: pointer;
+  display: flex;
+  gap: var(--sema-space-200, 8px);
+  height: 48px;
+  padding: var(--Positive-space-300, 12px) var(--Positive-space-1100, 44px)
+    var(--Positive-space-300, 12px) var(--Positive-space-400, 16px);
+  width: 343px;
+}
+
+.darkMode {
+  background: var(--sema-color-background-default, #000);
+  border: 1px solid var(--sema-color-border-interactive, #c2c2b6);
```

Comment:
let's remove this 

#c2c2b6)

it should just be `var(--sema-color-border-interactive)`